mysql-test/suite/galera/t/galera_gtid_slave_sst_rsync.test

```diff
@@ -169,26 +185,31 @@ DROP TABLE t2,t1;
 #
 
 --sleep 3
+
 --echo #Connection 2
 --connection node_2
 --let $wait_condition = SELECT COUNT(*) = 0 FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 't2';
 --source include/wait_condition.inc
+
 --echo #Connection 1
 --connection node_1
 --let $wait_condition = SELECT COUNT(*) = 0 FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 't1';
 --source include/wait_condition.inc
+
 --echo #Connection 2
 --connection node_2
 STOP SLAVE;
 RESET SLAVE ALL;
-set global wsrep_on=OFF;
-reset master;
-set global wsrep_on=ON;
+SET GLOBAL wsrep_on=OFF;
+RESET MASTER;
+SET GLOBAL wsrep_on=ON;
+
```

Comment:
Actually just changing the case is not a good idea , it creates a wrong git blame , SO I would suggest just keeping the same case